packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -13,9 +14,8 @@ import 'tabs.dart';
 /// the [TabBar.indicator].
 ///
 /// Descendant widgets obtain the current theme's [TabBarTheme] object using
-/// `Theme.of(context).tabBarTheme`.
-/// [ThemeData.tabBarTheme] can be customized by copying it (using
-/// [TabBarTheme.copyWith]).
+/// `TabBarTheme.of(context)`. Instances of [TabBarTheme] can be customized by
```

Comment:
can be customized with [TabBarTheme.copyWith][.

Reply:
Done.